ui/components/DashboardComponent.js

```diff
@@ -365,7 +365,7 @@ class DashboardComponent extends React.Component {
           }
         }
       },
-      self.handleError("Unable to fetch mesh scan data.")
+      self.redirectErrorToConsole("Unable to fetch mesh scan data.")
```

Comment:
No need for a new function if not widely used, change this to
```js
    this.props.enqueueSnackbar("Unable to fetch mesh scan data", {
      variant: "error",
      action: (key) => (
        <IconButton key="close" aria-label="Close" color="inherit" onClick={() => self.props.closeSnackbar(key)}>
          <CloseIcon />
        </IconButton>
      ),
      autoHideDuration: 7000,
    });
```
As we need to show a toaster for failed scan

Reply:
So I should not let the meshsync fail silently and instead show "Unable to fetch mesh scan data" for every error?

Reply:
Pardon for my wrong, just do console.log(err) and nothing else.